examples/simple_instance/main.tf

```diff
@@ -65,7 +65,7 @@ module "gce-container" {
 
 resource "google_compute_instance" "vm" {
   project      = var.project_id
-  name         = var.instance_name
+  name         = format("%s-%s", var.instance_name, md5(module.gce-container.container.image))
```

Comment:
don't repeat same code twice rather create local variable and use it

Reply:
Sure, fixed [here](https://github.com/terraform-google-modules/terraform-google-container-vm/compare/5f4e3ad7c0b3705e551760e363d3a2d5c0d1fb03..dfc3d20f1308c7f603f5208c8499062a23a13987)
Thank you for review, @kopachevsky 
Would you mind giving it another look please?